src/NUnitTestAdapterTests/Filtering/TestFilterTests.cs

```diff
@@ -121,12 +158,30 @@ public void PropertyValueProviderCategoryFail()
         [TestCase("NS11", new[] { "nUnitClassLibrary.NestedClasses.NC11" })]
         [TestCase("NS2", new[] { "nUnitClassLibrary.NestedClasses+NestedClass2.NC21" })]
         [TestCase("NS21", new[] { "nUnitClassLibrary.NestedClasses+NestedClass2.NC21" })]
-        public static void CanFilterConvertedTestsByCategory(string category, IReadOnlyCollection<string> expectedMatchingTestNames)
+        public void CanFilterConvertedTestsByCategoryWithTestCategoryAsFilter(string category, IReadOnlyCollection<string> expectedMatchingTestNames)
```

Comment:
Why you remove my static modifier? 😭 :D
Don't really care. `static` makes it more likely that people will choose to write more functional, parallel-safe tests.